src/components/SplitButton/__tests__/splitButton-tests.jest.js

```diff
@@ -1,16 +1,29 @@
 import React from "react";
+import "@testing-library/jest-dom";
```

Comment:
I did not see it here (if I missed it let me know) - let's add snapshot with split button menu and a snapshot with custom child (for making sure we are not breaking this ability now or in the fututre)